apps/forms-flow-ai/forms-flow-web/src/components/FOI/Header/FOIHeader.js

```diff
@@ -14,148 +14,192 @@ import {
 } from "../../../apiManager/services/FOI/foiNotificationServices";
 import {isMinistryLogin, getMinistryCode} from "../../../helper/FOI/helper";
 import io from "socket.io-client";
-import {SOCKETIO_CONNECT_URL, SOCKETIO_RECONNECTION_DELAY, SOCKETIO_RECONNECTION_DELAY_MAX} from "../../../constants/constants";
-import { fetchFOIFullAssignedToList } from "../../../apiManager/services/FOI/foiMasterDataServices";
-
-
-const FOIHeader = React.memo(({unauthorized=false}) => { 
+import {
+  SOCKETIO_CONNECT_URL,
+  SOCKETIO_RECONNECTION_DELAY,
+  SOCKETIO_RECONNECTION_DELAY_MAX,
+} from "../../../constants/constants";
 
-const dispatch = useDispatch(); 
-const isAuthenticated = useSelector((state) => state.user.isAuthenticated);
-const user = useSelector((state) => state.user.userDetail);
-let isMinistry = false;
-let ministryCode ="";
-const [screenPosition, setScreenPosition] = useState(0);
-const [open, setOpen] = useState(false);
-const closeModal = () => setOpen(false);
-const openModal = (coordinates) => {
-  const screenX = coordinates.pageX;
-  setScreenPosition(screenX);
-  setOpen(!open);
-}
-const [messageData, setMessageData] = useState([]);
-let foiNotifications = useSelector(state=> state.notifications.foiNotifications);
-const [socket, setSocket] = useState(null);
+const FOIHeader = React.memo(({ unauthorized = false }) => {
+  const dispatch = useDispatch();
+  const isAuthenticated = useSelector((state) => state.user.isAuthenticated);
+  const user = useSelector((state) => state.user.userDetail);
+  let isMinistry = false;
+  let ministryCode = "";
+  const [screenPosition, setScreenPosition] = useState(0);
+  const [open, setOpen] = useState(false);
+  const closeModal = () => setOpen(false);
+  const openModal = (coordinates) => {
+    const screenX = coordinates.pageX;
+    setScreenPosition(screenX);
+    setOpen(!open);
+  };
+  const [messageData, setMessageData] = useState([]);
+  let foiNotifications = useSelector(
+    (state) => state.notifications.foiNotifications
+  );
+  const [socket, setSocket] = useState(null);
 
-const userGroups = user?.groups?.map(group => group.slice(1));
-isMinistry = isMinistryLogin(userGroups);
-ministryCode = getMinistryCode(userGroups);
+  const userGroups = user?.groups?.map((group) => group.slice(1));
+  isMinistry = isMinistryLogin(userGroups);
+  ministryCode = getMinistryCode(userGroups);
 
-useEffect(() => {     
-  if(!unauthorized && isAuthenticated){
-    dispatch(fetchFOIFullAssignedToList());
```

Comment:
@aparna-aot added this line for showing the full name in notification. @aparna-aot - can you review this? do we need this line?

Reply:
Yeah. @jadmsaadaot.That line was added after there was issue in displaying the author names of notifications after the latest dashboard updates. So don't remove it.